roles/kubernetes-apps/helm/tasks/main.yml

```diff
@@ -2,18 +2,34 @@
 - name: Helm | Make sure HELM_HOME directory exists
   file: path={{ helm_home_dir }} state=directory
 
-- name: Helm | Set up helm launcher
+- name: Helm | Remove helm
+  file:
+    path: "{{ bin_dir }}/helm"
+    state: absent
+
+- name: Helm | Set up docker helm launcher
   template:
     src: helm-container.j2
     dest: "{{ bin_dir }}/helm"
     owner: root
     mode: 0755
-  register: helm_container
+  when: helm_deployment_type == "docker"
+
+- name: Helm | Set up binary helm launcher
```

Comment:
It's probably worth adding a fact in here to see what version of helm is currently installed